.eslintrc.js

```diff
@@ -1,13 +1,23 @@
+'use strict';
+
+/**
+ * @file    Manages the ESLint rules
+ * @author  Fimion
+ */
+
 module.exports = {
   extends: [
     'tjw-base',
-    'tjw-import'
+    'tjw-import',
+    'tjw-jsdoc'
   ],
   globals: {
     Promise: true,
     Proxy: true,
     Reflect: true
   },
   rules: {
+    'jsdoc/check-examples': 'off',
+    'jsdoc/require-example': 'off'
```

Comment:
I'm pulling in my own JSDoc ruleset that I use in my repos. I turned off these two as they are more tedious than useful.

My ruleset is here:

* https://github.com/tjw-lint/eslint-config-tjw-jsdoc/blob/main/index.js

if you want to tweak any of it, or use it as a jumping off point for your own set.

Official docs are here:

* https://github.com/gajus/eslint-plugin-jsdoc

I found a few things that I wish they did differently, so I made issues for those things and with in a day or so a new release came out with those features added. So don't be afraid to reach out to them, very nice people 😃 👍